src/Maestro/Maestro.MergePolicyEvaluation/MergePolicyEvaluationResults.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+
+#nullable enable
```

Comment:
Are these needed since you enabled this for the whole project?

Reply:
They're not, didn't notice it was enabled by default